### ISSUE TITLE
[total_height] follows the last uploaded file, NOT the file that is being printed.

### ISSUE THREAD

**Issue by ruedli** [OllisGit/OctoPrint-DisplayLayerProgress#220]
When during a print, I upload a new job, all statistics remains as active for the printing job, however [total_height] now disdplyas the height of the NEW uploaded file, not the one that is being printed.

I see this value both in the window title bar, as on the dashboard.

Using  DisplayLayerProgress 1.25.4
Octoprint: 

browser.user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36
connectivity.connection_check: 8.8.8.8:53
connectivity.connection_ok: true
connectivity.enabled: true
connectivity.online: true
connectivity.resolution_check: octoprint.org
connectivity.resolution_ok: true
env.hardware.cores: 4
env.hardware.freq: 1500
env.hardware.ram: 1902264320
env.os.bits: 32
env.os.id: linux
env.os.platform: linux
env.plugins.pi_support.model: Raspberry Pi 4 Model B Rev 1.4
env.plugins.pi_support.octopi_version: 0.18.0
env.plugins.pi_support.throttle_state: 0x0
env.python.pip: 20.3.3
env.python.version: 3.7.3
env.python.virtualenv: true
octoprint.safe_mode: false
octoprint.version: 1.5.3
printer.firmware: Prusa-Firmware 3.9.2 based on Marlin

![image](https://user-images.githubusercontent.com/5008440/113178011-6f756100-924e-11eb-9112-61cde27b2261.png)

![image](https://user-images.githubusercontent.com/5008440/113178145-916ee380-924e-11eb-9627-25810eb422d5.png)

**Comment by ruedli** [comment]
I also use ARC-Welder, which processes the file during print. Not sure if this contributes to the problem

**Comment by OllisGit** [comment]
Hi @ruedli,

how do you upload the file? Via Browser Drag&Drop, Upload-Button or from Sclicer (api-call)?

**Comment by OllisGit** [comment]
Hi @ruedli,

new release 1.26.0 is out, please test and give feedback.

Thx, in advance
Olli

**Comment by ruedli** [comment]
Hi Olli, @OllisGit ,

I downloaded the new release, did one test and the problem seems resolved. My test sequence was:

- Print a file with height of 6.2mm
- While the printer is heating up: upload a file with height of 78mm.
- The print got processed by arc-welder and was processed for its internal parameters (like filament usage). The "detailed" information, as well as the new thumbnail became visible (while the printer was still heating up).
- The [total_height] was NOT updated, so this FIXED the issue I had seen before, where [total_height] was replaced by the value of the new uploaded file.

From my perspective it is OK now, thnx for fixing this!

Ruedli
